instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
Dear Marnach,

I have been mentioned in this forum due to a spelling mistake, please stop
mentioning me :)

Thank you.

On Wed, Apr 27, 2016 at 1:06 AM, Sven Marnach <notifications@github.com>
wrote:

> In instance/templates/instance/ansible/swift.yml
> <https://github.com/open-craft/opencraft/pull/54#discussion_r61173246>:
>
> > @@ -0,0 +1,6 @@
> > +EDXAPP_SWIFT_AUTH_VERSION: '2'
> > +EDXAPP_SWIFT_USERNAME: '{{ user }}'
> > +EDXAPP_SWIFT_KEY: '{{ password }}'
> > +EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
> > +EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
> > +EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
>
> @omarkhan <https://github.com/omarkhan> Thanks for the suggestion. This
> sounds like it will work. I will test it tomorrow.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/open-craft/opencraft/pull/54/files/6a8821888d1aba92085204eed62b6f09114a97f2#r61173246>
>